core-bundle/src/Command/MigrateCommand.php

```diff
@@ -71,7 +71,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             if ($errors = $this->compileConfigurationErrors()) {
                 if ($asJson) {
                     foreach ($errors as $message) {
-                        $this->writeNdjson('error', ['message' => $message]);
+                        $this->writeNdjson('problem', ['message' => $message]);
```

Comment:
I would prefer `database-error` instead of `problem` because I don‘t want to end up with `error`, `problem`, `fault`, `issue`, `failure` and more synonyms in case we have to add more error categories in the future. 😉

Reply:
we've extensively discussed these options in the public call and agreed that `problem` would be sufficient 🤷

Reply:
But wouldn’t `database-error` be sufficient as well?

Reply:
anything would be sufficient, `foobar` would work as well as long as the Manager knows about it 🤷 
To me
 - there was an **error** executing the script
 - there is a **problem** in your database (there is no error in your DB, but a problem in the config!)
 - there is a **warning** about your database (or something else, but the app works)